object-manipulation-3/main.js

```diff
@@ -0,0 +1,57 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var players = [
+  { name: 'paul', firstHand: null, secondHand: null },
+  { name: 'denny', firstHand: null, secondHand: null },
+  { name: 'heesoo', firstHand: null, secondHand: null },
+  { name: 'isaac', firstHand: null, secondHand: null }
+];
+var deck = [];
+
+function playGame() {
+
+  function createDeck() {
+    var suits = ['spades', 'diamond', 'heart', 'club'];
+    var name = [
+      ['A', 11],
+      ['2', 2],
+      ['3', 3],
+      ['4', 4],
+      ['5', 5],
+      ['6', 6],
+      ['7', 7],
+      ['8', 8],
+      ['9', 9],
+      ['10', 10],
+      ['J', 10],
+      ['Q', 10],
+      ['K', 10]
+    ];
+    for (let i = 0; i < suits.length; i++) {
+      for (let j = 0; j < name.length; j++) {
+        var card = { rank: name[j], suit: suits[i] };
+        deck.push(card);
+      }
+    }
+  }
+
+  createDeck();
+  var shuffleDeck = _.shuffle(deck);
+  for (let i = 0; i < players.length; i++) {
+    players[i].firstHand = shuffleDeck[Math.floor(Math.random() * 52)];
+    players[i].secondHand = shuffleDeck[Math.floor(Math.random() * 52)];
```

Comment:
Instead of grabbing a random card from the deck and not removing it, you should make this function like a real card game. Remove cards from the top of the deck, and actually remove them so other players can't have the same exact card